docs/cvl/builtin.md

```diff
@@ -4,7 +4,84 @@ Built-in Rules
 The Prover has some built-in general-purpose rules that can be verified on a
 contract out-of-the-box.
 
+## Deep Sanity
+
+Enable with:
+```cvl
+use builtin rule deepSanity;
+```
+
+Ensure `--multi_assert_check` is enabled (will throw an error otherwise!).
```

Comment:
replace with
Ensure `--multi_assert_check` is enabled (otherwise, it will throw an error).